operational_analysis/types/reanalysis.py

```diff
@@ -23,10 +25,53 @@ def __init__(self, engine="pandas"):
             self._sc = self._pyspark.SparkContext.getOrCreate()
             self._sqlContext = self._sql.SQLContext.getOrCreate(self._sc)
 
-    def load(self, path, name):
+    def load(self, path, name, format="csv", lat=None, lon=None, **kwargs):
+        """
+        This method loads reanalysis data into the ReanalysisData object using one of two formats. When format is
+        "csv", reanalysis data will be imported from csv files in the specified path with file name "{name}_{product}.
+        csv", where product is the reanalysis product name. When format is "planetos", data will be imported from csv
+        files if they exist in the specified path. Otherwise, reanalysis data will be downloaded via the PlanetOS API,
+        saved as csv files in the specified path, and then loaded into the ReanalysisData object. Note that only
+        "merra2" and "era5" reanalysis products are supported when using the "planetos" format.
+
+        Args:
+            path (:obj:`string`): Path where reanalysis files are or will be saved
+            name (:obj:`string`): String used as a prefix when forming reanalysis data file names (e.g., the
+                project name).
+            format (:obj:`string`, optional): Format of the reanalysis data. If "csv", data will be imported from
+                existing csv files. If "planetos", data will be downloaded via the PlanetOS API. Defaults to "csv".
+            lat (:obj:`float`, optional): Latitude (degrees). Used when format = "planetos". Defaults to None.
+            lon (:obj:`float`, optional): Longitude (degrees). Used when format = "planetos". Defaults to None.
+            **kwargs: Optional keyword arguments passed to
+                :py:func:`operational_analysis.toolkits.reanalysis_downloading.download_reanalysis_data_planetos`
+
+        Raises:
+            NotImplementedError: When a format other than "csv" or "planetos" is specified
+            NotImplementedError: When the ReanalysisData object's engine is specified as "spark"
+        """
+
         if self._engine == "pandas":
-            for product in self._products:
-                self._product[product].load(path, "{}_{}".format(name, product))
+            if format == "csv":
+                for product in self._products:
+                    self._product[product].load(path, "{}_{}".format(name, product))
+            elif format == "planetos":
+                for product in [p for p in self._products if p in ["merra2", "era5"]]:
+                    # Download from PlanetOS if csv file doesn't already exist
+                    if not os.path.exists(os.path.join(path, "{}_{}.csv".format(name, product))):
```

Comment:
I know f-strings haven't quite caught on, but it does help to make a path string like this a little bit easier to read:
```python
if not os.path.exists(os.path.join(path, f"{name}_{product}.csv"))
```
and if you took it one step further with a [`pathlib.Path`](https://docs.python.org/3.8/library/pathlib.html#correspondence-to-tools-in-the-os-module) object:
```python
if not (Path(path) / f"{name}_{product}.csv").exists():
```

Reply:
Done